webapp/src/js/services/target-aggregates.js

```diff
@@ -0,0 +1,257 @@
+const moment = require('moment');
+const _ = require('lodash');
+
+angular.module('inboxServices').factory('TargetAggregates',
+  function(
+    $q,
+    Auth,
+    CalendarInterval,
+    ContactTypes,
+    DB,
+    GetDataRecords,
+    Search,
+    Settings,
+    UHCSettings,
+    UserSettings
+  ) {
+
+    'use strict';
+    'ngInject';
+
+    const getCurrentIntervalTag = (settings) => {
+      const uhcMonthStartDate = UHCSettings.getMonthStartDate(settings);
+      const targetInterval = CalendarInterval.getCurrent(uhcMonthStartDate);
+
+      return moment(targetInterval.end).format('Y-MM');
```

Comment:
If I'm reading this right it means you can only ever see targets for the current month. It would be very useful to be able to see the targets for the previous month as the targets may not have even replicated to the supervisor until the month has ended. Also there is a common workflow for supervisors to sit down with the CHW a few days after the end of the month to check up on how everything's going.

What do you think?

It's a bit late in the piece to be bringing up new functionality like this so maybe it's worth discussing with design and product owners and raising a new issue to add that functionality in a later release?

Reply:
This has been very much the focus of many conversations and is already mentioned as a bullet point [here](https://github.com/medic/cht-core/issues/4840#issuecomment-578224608)

Reply:
Yeah I think it's fine for now. We'll hear pretty quickly if we're missing an important use case...